activesupport/lib/active_support/values/time_zone.rb

```diff
@@ -94,7 +94,7 @@ class TimeZone
       "Bucharest"                    => "Europe/Bucharest",
       "Cairo"                        => "Africa/Cairo",
       "Helsinki"                     => "Europe/Helsinki",
-      "Kyiv"                         => "Europe/Kiev",
+      "Kyiv"                         => "Europe/Kyiv",
```

Comment:
IANA timezone identifier is `Europe/Kiev` and you just can't change that in Rails. Please contact IANA first: https://www.iana.org/time-zones

Reply:
It was fixed in version 2022b https://mm.icann.org/pipermail/tz-announce/2022-August/000071.html